code/mob/living/carbon/human/procs/emote.dm

```diff
@@ -230,6 +230,18 @@
 		#ifdef DATALOGGER
 						game_stats.Increment("farts")
 		#endif
+				if(src.mutantrace && src.mutantrace.name == "dwarf" && prob(3))
+					var/glowsticktype = pick(/obj/item/device/light/glowstick,
+					/obj/item/device/light/glowstick/white,/obj/item/device/light/glowstick/yellow,
+					/obj/item/device/light/glowstick/blue,/obj/item/device/light/glowstick/purple,
+					/obj/item/device/light/glowstick/pink,/obj/item/device/light/glowstick/cyan,
+					/obj/item/device/light/glowstick/orange,/obj/item/device/light/glowstick/red)
```

Comment:
why is this a manual list and not just typesof()

Reply:
that.........exists? :D